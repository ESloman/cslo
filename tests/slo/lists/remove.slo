var x = [1, 2, 3];
println(x);

println("Removed: ", x.remove(0));
println(x);

var y = x.remove(1);
println("Removed: ", y);
println(x);
