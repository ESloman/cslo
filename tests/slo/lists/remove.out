list[3]: [1, 2, 3]
Removed: 1
list[2]: [2, 3]
Removed: 3
list[1]: [2]
